src/Groups/cosets.jl

```diff
@@ -457,22 +457,42 @@ end
 Group double coset.
 Two double cosets are equal if, and only if, they contain the same elements.
 """
-struct GroupDoubleCoset{T <: GAPGroup, S <: GAPGroupElem}
+mutable struct GroupDoubleCoset{T <: GAPGroup, S <: GAPGroupElem}
 # T=type of the group, S=type of the element
    G::T
    H::GAPGroup
    K::GAPGroup
    repr::S
    X::GapObj
+   size
```

Comment:
Either do
```suggestion
   size::GapInt
```
or
```suggestion
   size::ZZRingElem
```
I tend to prefer the latter.